sagemaker_processing/scikit_learn_data_processing_and_model_evaluation/scikit_learn_data_processing_and_model_evaluation.ipynb

```diff
@@ -46,6 +46,11 @@
     "import sagemaker\n",
```

Comment:
<p>Should move any pip installs to a prerequisites section at the top.</p><p>Also use <code>%pip install ...</code>  instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1762/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>